opengrok-indexer/src/main/java/org/opengrok/indexer/configuration/Configuration.java

```diff
@@ -807,28 +806,6 @@ public void setHistoryCache(boolean historyCache) {
         this.historyCache = historyCache;
     }
 
-    /**
-     * How long can a history request take before it's cached? If the time is
-     * exceeded, the result is cached. This setting only affects
-     * {@code FileHistoryCache}.
-     *
-     * @return the maximum time in milliseconds a history request can take
-     * before it's cached
-     */
-    public int getHistoryCacheTime() {
-        return historyCacheTime;
```

Comment:
Maybe also remove the `historyCacheTime` field?

Isn't this going to break the deserialization if the field is present in XML?

Reply:
yes and yes, will add a note to the release.